prime-router/src/main/kotlin/azure/DatabaseAccess.kt

```diff
@@ -237,6 +241,14 @@ class DatabaseAccess(private val create: DSLContext) {
             config.addDataSourceProperty("cachePrepStmts", "true")
             config.addDataSourceProperty("prepStmtCacheSize", "250")
             config.addDataSourceProperty("prepStmtCacheSqlLimit", "2048")
+            // See this info why these are a good value
+            //  https://github.com/brettwooldridge/HikariCP/wiki/About-Pool-Sizing
+            config.minimumIdle = 2
+            config.maximumPoolSize = 8
+            // This strongly recommended to be set "be several seconds shorter than any database or infrastructure
+            // imposed connection time limit". Not sure what value is but have observed that connection are closed
+            // after about 10 minutes
+            config.maxLifetime = 180000
```

Comment:
I've been meaning to check whether there's a simple way to just have connections stay open "forever" when I'm running in the debugger locally.    Connections closing make the debugging more confusing, especially if you are trying to debug something database related!